### DISCUSSION TITLE
Kushal theme triggers notification on Kitty terminal

### DISCUSSION THREAD

**Discussion by joaociocca** [JanDeDobbeleer/oh-my-posh#5303] [Troubleshoot]
When I use the [kushal theme](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/kushal.omp.json) with [kitty terminal](https://sw.kovidgoyal.net/kitty/), a weird `9;${CWD}` desktop notification shows up.

![image](https://github.com/user-attachments/assets/6bd7b81e-d3c5-4fd2-be8c-add14521889f)

If I switch to [tokyo theme](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/tokyo.omp.json), this notification doesn't pop up.

Kitty triggers these notifications by `printf '\x1b]99;;Hello world\x1b\\'`, according to the [documentation](https://sw.kovidgoyal.net/kitty/desktop-notifications/), but I can't figure out where to fix this...?

**Comment by JanDeDobbeleer** [comment]
It's `pwd: OSC99`

**Reply by joaociocca** [reply]
I can see `"osc99": true,` in there, but removing that line or changing to false does nothing...?

**Reply by JanDeDobbeleer** [reply]
removing that part will stop it from printing that sequence. That should work, or you are editing the wrong configuration. Because that's the origin.

**Reply by joaociocca** [reply]
damn, I feel kinda stupid now. Yeah, I was editing the same file but in a different place than I was loading from when running the eval :facepalm:

**Reply by JanDeDobbeleer** [reply]
@joaociocca we've all been there. Nothing to be ashamed of!
